### PR TITLE
docs: add play section

### PR DESCRIPTION
**Pull request by sunng87**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Add GreptimePlay section in our readme. 

@xtang @apdong2022 You may want add better text copy for Play. [Rendered](https://github.com/GreptimeTeam/greptimedb/blob/chore/add-play/README.md)

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,13 @@ for years. Based on their best-practices, GreptimeDB is born to give you:
 
 ## Quick Start
 
+### GreptimePlay
+
+GreptimePlay is a playground built on managed GreptimeDB, where you can try out
+GreptimeDB features from browser.
+
+<a href="https://greptime.com/playground" target="_blank"><img src="https://www.greptime.com/assets/greptime_play_button_colorful.1bbe2746.png" alt="GreptimePlay" /></a>
```
Comment (sunng87) [inline README.md:55]: @alili is the image link permanent between our website deployments?

File: README.md [hunk 2]
```diff
@@ -47,6 +47,15 @@ for years. Based on their best-practices, GreptimeDB is born to give you:
 
 ## Quick Start
 
+### GreptimePlay
+
+GreptimePlay is a playground built on managed GreptimeDB, where you can try out
```
Comment (killme2008) [inline README.md:52]:
```suggestion
GreptimePlay is a playground built on fully managed GreptimeDB, where you can try out
```

**Review by killme2008 — APPROVED** [review]
LGTM

**Review by killme2008 — APPROVED** [review]
LGTM

**Comment by killme2008** [comment]
@xtang PTAL

File: README.md [hunk 3]
```diff
@@ -47,6 +47,14 @@ for years. Based on their best-practices, GreptimeDB is born to give you:
 
 ## Quick Start
 
+### GreptimePlay
+
+Try out GreptimeDB features right from your browser.
```
Comment (xtang) [inline README.md:52]:
```suggestion
Try out the features of GreptimeDB right from your browser.
```

**Pull request merged**